parmed/amber/parameters.py

```diff
@@ -264,23 +269,52 @@ def from_leaprc(cls, fname, search_oldff=False):
             own_handle = False
         # To make parsing easier, and because leaprc files are usually quite
         # short, I'll read the whole file into memory
-        lines = map(lambda line:
-                line if '#' not in line else line[:line.index('#')], f)
+        def joinlines(lines):
+            newlines = []
+            composite = []
+            for line in lines:
+                if line.endswith('\\\n'):
+                    composite.append(line[:-2])
+                    continue
+                else:
+                    composite.append(line)
+                newlines.append(''.join(composite))
+                composite = []
+            if composite:
+                newlines.append(''.join(composite))
+            return newlines
+        lines = joinlines(map(lambda line:
+                    line if '#' not in line else line[:line.index('#')], f))
         text = ''.join(lines)
         if own_handle: f.close()
         lowertext = text.lower() # commands are case-insensitive
         # Now process the parameter files
-        for fname in _loadparamsre.findall(text):
-            if fname[0] in ('"', "'"): fname = fname[1:-1]
-            fname = fname.replace(r'\ ', ' ')
-            params.load_parameters(_find_amber_file(fname, search_oldff))
-        # Now process the library file
-        for fname in _loadoffre.findall(text):
-            if fname[0] in ('"', "'"): fname = fname[1:-1]
-            fname = fname.replace(r'\ ', ' ')
-            params.residues.update(
+        def process_fname(fname):
+            if fname[0] in ('"', "'"):
+                fname = fname[1:-1]
+            fname = fname.replace('_BSTOKEN_', r'\ ').replace(r'\ ', ' ')
+            return fname
+        for line in lines:
+            line = line.replace(r'\ ', '_BSTOKEN_')
+            if _loadparamsre.findall(line):
+                fname = process_fname(_loadparamsre.findall(line)[0])
+                params.load_parameters(_find_amber_file(fname, search_oldff))
+            elif _loadoffre.findall(line):
+                fname = process_fname(_loadoffre.findall(line)[0])
+                params.residues.update(
                     AmberOFFLibrary.parse(_find_amber_file(fname, search_oldff))
-            )
+                )
+            elif _loadmol2re.findall(line):
```

Comment:
Why would it have any affect on the coverage score?  `else: pass` is a no-op.  Even if I turned on branch coverage that branch would still be triggered.

Also, this logic shows the intent.  Each "if" handles a case that is processed from the leaprc file and everything else is ignored (like "loadPDB" and "solvateOct/Box").  As you support more commands (like `source` is on the docket next), then you just add another "elif" block.